vkFFT/vkFFT.h

```diff
@@ -35409,6 +35445,7 @@ static inline VkFFTResult VkFFTPlanAxis(VkFFTApplication* app, VkFFTPlan* FFTPla
 	axis->specializationConstants.warpSize = app->configuration.warpSize;
 	axis->specializationConstants.numSharedBanks = app->configuration.numSharedBanks;
 	axis->specializationConstants.useUint64 = app->configuration.useUint64;
+	axis->specializationConstants.useStrict32BitAddress = app->configuration.useStrict32BitAddress;
```

Comment:
This will break all other backends as you reference it not only in VKFFT_BBACKEND=2.

Reply:
Should be fixed now.